### DISCUSSION TITLE
Focus problem if OnChange action

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#11892] [Help]
I'm editing my form, passing from one field to another by hitting TAB key.
IF I have an OnChange event that fires on a field, then the focus leaves the next field.
(Focus appears one second, then disappear.)
Then I need to click inside the next field to put the focus on it, and be able to edit it.

I think it was working better a few weeks/month ago... Could it be some regression of the last releases ? Do you see a workaround ?

Thanks

**Comment by andz-bb** [comment]
hey @NicolasFabre81, which on change actions are you using on your form fields that are causing the focus to be lost? Which browser are you using and which version of Budibase? If you could send an app export with a recreate that would be really useful. Thanks
